src/test/Components/CreateImageWizardV2/CreateImageWizard.compliance.test.tsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+
+import '@testing-library/jest-dom';
+
+import { screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import CreateImageWizard from '../../../Components/CreateImageWizard/CreateImageWizard';
```

Comment:
The tests are currently testing the old wizard, this will need to be updated like this:
```suggestion
import CreateImageWizard from '../../../Components/CreateImageWizardV2/CreateImageWizard';
```

It should then make some tests fail. Some parts of the tests can be commented out as they are not relevant yet (like checking for the "File system configuration" header and so).